### PR TITLE
Migrating tabbed view as example of how migration could look like

### PR DESCRIPTION
**Pull request by megoth**
Want to show Tim how I'm thinking this can be done in general

### CODE REVIEW

File: src/tabbed/tabbedPane.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        renderMain: prepareRenderMainFn(context),
+        renderTab: prepareRenderTabFn(subject, context),
+        backgroundColor: store.anyValue(subject, ns.ui('backgroundColor')) || '#ddddcc'
+      }))
+    })()
+    return div
+  }
+}
+
+function prepareRenderTabFn (subject: NamedNode, context: DataBrowserContext): (div, item) => void {
```
Comment (timbl) [inline src/tabbed/tabbedPane.ts]: As a question of code style, simpler is better.   A one-line `renderTab` in line where it has its state used is better than a four-line one far from where it is used.    A function is simpler than a function generator function.

**Review by timbl — CHANGES_REQUESTED** [review]
The changes to ts seem fine in general.  Suggest that render() be simplified back to simpler functions as it had been.  (Are we sure we want to use export default?)

**Comment by megoth** [comment]
@timbl Have implemented your requested changes, hopefully it looks good now ^_^

File: src/tabbed/tabbedPane.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+function prepareRenderMainFn (context: DataBrowserContext): (containerDiv, item) => void {
+  return (containerDiv, item) => {
+    containerDiv.innerHTML = ''
+    const table = containerDiv.appendChild(context.dom.createElement('table'))
+    ;(context.getOutliner(context.dom) as any).GotoSubject(item, true, null, false, undefined, table)
+  }
+}
+
+export default TabbedPane
```
Comment (timbl) [inline src/tabbed/tabbedPane.ts:58]: Are you sure that export default is what we need -- won't that give is a .default. in the path when we use a require()  ?

**Review by timbl — CHANGES_REQUESTED** [review]

File: src/tabbed/tabbedPane.ts (hunk 2 above)
Comment (megoth) [reply src/tabbed/tabbedPane.ts:58]: I forgot to note that this PR is dependent on the changes in https://github.com/solid/solid-panes/pull/202#discussion_r359061029 - with it we will support both CJS and ES modules.

**Comment by megoth** [comment]
@timbl Am I to understand that you approve the rest of this PR now that https://github.com/solid/solid-panes/pull/202 is merged?

**Pull request merged**